doc/content/getting_started/NEML2.md

```diff
@@ -0,0 +1,68 @@
+# NEML2
+
+BlackBear offers a wide variety of material models. In addition to the native material models, BlackBear can also seamlessly interface with an external material modeling library called [NEML2](https://github.com/reverendbedford/neml2).
+
+## What is NEML2?
+
+NEML2 stands for the New Engineering Material model Library, version 2.
```

Comment:
Is N for New or Nuclear?

Reply:
IIRC that's our unsuccessful attempt at rebranding NEML2 to deemphasize "nuclear"...